README.md

```diff
@@ -50,42 +49,41 @@
 
```

Comment:
![image](https://github.com/katfogy/vet-clinic/assets/93039359/fb56a176-512a-4d3d-bb84-226987b305bf)

**[OPTIONAL]**

- [ ] It would be nice if you make all the commit messages capitlized to make your pull request professional and descriptive 😊